site/src/@ndlib/gatsby-theme-marble/components/Pages/MarbleItem/ItemLayout/TombstoneMetadata/test.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import TombstoneMetadata from './'
+
+describe('TombstoneMetadata', () => {
+  test('Creator value is null', () => {
+    const item = {
+      id: 'ABC123',
+      description: 'How the Grinch Stole Christmas',
+      metadata: [
+        {
+          label: 'Creator',
```

Comment:
Just so you know, I believe this scenario won't happen.  Either the Creator node will contain a display field with will either contain "unknown" or some valid value, or the Creator node will not exist in the metadata.

Reply:
Thank you, that changes things a little bit. Fixing to reflect